src/index.js

```diff
@@ -113,68 +124,68 @@ export default class ModalFilterPicker extends Component {
   renderOptionList = () => {
     const {
       noResultsText,
-      listViewProps,
+      flatListViewProps,
+      keyExtractor,
       keyboardShouldPersistTaps
     } = this.props
 
     const { ds } = this.state
 
-    if (1 > ds.getRowCount()) {
+    if (!ds.length) {
       return (
-        <ListView
-          enableEmptySections={false}
-          {...listViewProps}
-          dataSource={ds.cloneWithRows([{ key: '_none' }])}
-          renderRow={() => (
+        <FlatList
+          data={ds}
+          keyExtractor={keyExtractor || this.constructor.keyExtractor}
+          {...flatListViewProps}
+          renderItem={() => (
             <View style={styles.noResults}>
               <Text style={styles.noResultsText}>{noResultsText}</Text>
             </View>
           )}
         />
       )
-    } else {
-      return (
-        <ListView
-          enableEmptySections={false}
-          {...listViewProps}
-          dataSource={ds}
-          renderRow={this.renderOption}
-          keyboardShouldPersistTaps={keyboardShouldPersistTaps}
-        />
-      )
     }
+    return (
+      <FlatList
+        keyExtractor={keyExtractor || this.keyExtractor}
```

Comment:
Needs to be `this.constructor.keyExtractor`

Reply:
Ah! Thanks for pointing out. It's fixed now @hiddentao